### PR TITLE
:seedling: Add version information directly from git

### PR DESCRIPTION
**Pull request by oz123**
Partial fix for kairos-io/kairos#643.

Signed-off-by: Oz Tiram <oz@spectrocloud.com>

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -79,9 +79,13 @@ BUILD_GOLANG:
     ARG CGO_ENABLED
     ARG BIN
     ARG SRC
+    COPY +version/VERSION ./
+    ARG VERSION=$(cat VERSION)
     ENV CGO_ENABLED=${CGO_ENABLED}
-
-    RUN go build -ldflags "-s -w" -o ${BIN} ./cmd/${SRC} && upx ${BIN}
+    ARG LDFLAGS="-s -w -X 'github.com/kairos-io/kairos/pkg/common.VERSION=$VERSION'"
```
Comment (mudler) [inline Earthfile:85]: There is no need to expose this in `pkg`, so better to consume a var inside `internal`. Also use camelcase here, I think the linter will complain

Comment (oz123) [reply Earthfile:85]: Ah, thanks for reminding me about `internal`!  I should use that, indeed!

Comment (oz123) [reply Earthfile]: Done.

**Review by mudler — DISMISSED** [review]

File: cmd/profile-build/main.go [hunk 2]
```diff
@@ -12,7 +13,7 @@ func main() {
 
 	app := &cli.App{
 		Name:    "profile-build",
-		Version: "0.1",
+		Version: common.VERSION,
 		Author:  "Ettore Di Giacinto",
```
Comment (mudler) [inline cmd/profile-build/main.go:17]:
```suggestion
		Author:  "Kairos authors",
```

**Review by mudler — APPROVED** [review]

**Pull request merged**